### PR TITLE
Feature/e UI 8457

### PR DESCRIPTION
**Pull request by MateiHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-8457

### Change description ###
Fix spinner display behavior on case edit

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-edit-page/case-edit-page.component.spec.ts [hunk 1]
```diff
@@ -1133,7 +1151,7 @@ xdescribe('CaseEditPageComponent', () => {
       expect(text(firstFieldError)).toBe('First field error');
       const secondFieldError = fieldErrorList.query($SELECT_SECOND_FIELD_ERROR);
       expect(text(secondFieldError)).toBe('Second field error');
-      expect(comp.showSpinner).toBeFalsy();
+      // expect(comp.showSpinner).toBeFalsy();
```
Comment (DavidJayakumar) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-edit-page/case-edit-page.component.spec.ts:1154]: commented lines can be removed

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-edit-page/case-edit-page.component.spec.ts [hunk 2]
```diff
@@ -1055,9 +1073,10 @@ xdescribe('CaseEditPageComponent', () => {
       fixture.detectChanges();
 
       expect(eventData.case_reference).toBeUndefined();
-      expect(comp.showSpinner).toBeFalsy();
-
+      // expect(comp.showSpinner).toBeFalsy();
```
Comment (DavidJayakumar) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-edit-page/case-edit-page.component.spec.ts:1076]: commented lines can be removed

**Review by DavidJayakumar — CHANGES_REQUESTED** [review]
@MateiHMCTS Can the commented lines be removed?

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-edit-page/case-edit-page.component.ts [hunk 3]
```diff
@@ -19,6 +19,8 @@ import { CaseEditComponent } from '../case-edit/case-edit.component';
 import { WizardPage } from '../domain/wizard-page.model';
 import { Wizard } from '../domain/wizard.model';
 import { PageValidationService } from '../services/page-validation.service';
+import { LoadingService } from '../../../services';
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-edit-page/case-edit-page.component.ts:22]: @MateiHMCTS - please re-order the import statements (linting)

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@MateiHMCTS - just one very minor linting change required; the rest is OK

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@MateiHMCTS - just one quick question on the import order

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/linked-cases/write-linked-cases-field.component.spec.ts [hunk 4]
```diff
@@ -10,6 +10,8 @@ import { CaseEventData } from '../../../domain/case-event-data.model';
 import { CaseView } from '../../../domain/case-view';
 import { CaseField } from '../../../domain/definition/case-field.model';
 import { Draft } from '../../../domain/draft.model';
+import createSpyObj = jasmine.createSpyObj;
```
[the remark this answers is not in the document]
Comment (MateiHMCTS) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/linked-cases/write-linked-cases-field.component.spec.ts]: Amended; for some weird reason if I have a unordered import and I try to fix it with my IDE; it'll also pull that import createSpyObj line.

**Review by DavidJayakumar — APPROVED** [review]
LGTM

**Review by Daniel-Lam — APPROVED** [review]

**Pull request closed without merging**